agent/06_agent_create_cluster.sh

```diff
@@ -360,6 +355,11 @@ case "${AGENT_E2E_TEST_BOOT_MODE}" in
     ;;
 
   "DISKIMAGE" )
+    # Create the config ISO
+    mkdir -p ${config_image_dir}
+    cp ${asset_dir}/*.yaml ${config_image_dir}
+    create_config_image
```

Comment:
Could make sense to move these commands directly within the `create_config_image`? They anyhow prepare the input ingredients for the subsequent `create config-image` installer command

Reply:
It's actually intentionally separated, see logic for it in 'create_factory_image': https://github.com/openshift-metal3/dev-scripts//blob/4f8e2a93864a5ee6d613174413741812dc4e20d8/agent/06_agent_create_cluster.sh#L43